utils/mil_msgs/msg/LabeledObject.msg

```diff
@@ -0,0 +1,11 @@
+# Represents a single object labeled from LabelMe
+
+string name                   # Name/Type of object entered into labeling program
```

Comment:
Since this is in mil_common, no. Each project should have a wiki page / script on the usage of this for their project.